### PR TITLE
fix: managed node installation fixes contentful backup

### PR DESCRIPTION
**Pull request by SilasPeters**
This PR aims to solve the issue that the backup script cannot succesfully load the contentful-cli package to backup contentful data. This is caused by a too old version of nodejs. Thus, this PR actually fixes a way larger problem: properly maintain nodejs installations per ansible role.

It changes the following:

- Instead of a global Nodejs and yarn installation, a global nvm installation is maintained.
- Every role which needs to install any nodejs version, should be added to the `nvm` group.
- As a result, every ansible role who installs a nodejs version just has to simply source the global `nvm.sh` and then install and use the proper nodejs version - like one would have done without nvm.

**TLDR**; every role just has to source the `nvm.sh` script and then do what he would normally do. This is made possible by not installing nodejs globally but nvm.

### CODE REVIEW

**Review by TobiasDeBruijn — APPROVED** [review]

**Review by stickyPiston — DISMISSED** [review]
頑張ったね！

I can't really comment on the ansible code, but the use of nvm looks good.
